modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -184,8 +189,6 @@ body {
 }
 
 .apos-admin-bar__sub /deep/ .apos-context-menu__btn {
-  padding-left: 20px;
```

Comment:
Aligning the second row button set me off looking at this 20px value as well. These are overridden and not actually used, it turns out.